app/views/shared/_bugsnag_javascript.html.haml

```diff
@@ -1,3 +1,4 @@
+- # TODO REMOVE JS FROM HERE AND MOVE IT INTO ERROR_HANDLER.JS.COFFE
 - if ENV['BUGSNAG_JAVASCRIPT_API_KEY']
   %script{ src:"//d2wy8f7a9ursnm.cloudfront.net/bugsnag-2.min.js",
     "data-apikey" => "#{ENV["BUGSNAG_JAVASCRIPT_API_KEY"]}" }
```

Comment:
it is okay to have this script tag here, but not okay to have the rest of the JS. It should go in error_handler.js